indexing-hadoop/src/main/java/io/druid/indexer/HadoopDruidIndexerMapper.java

```diff
@@ -104,6 +105,10 @@ public final static InputRow parseInputRow(Object value, InputRowParser parser)
     if(parser instanceof StringInputRowParser && value instanceof Text) {
       //Note: This is to ensure backward compatibility with 0.7.0 and before
       return ((StringInputRowParser) parser).parse(value.toString());
+    } else if(parser instanceof StringInputRowParser && value instanceof BytesWritable) {
```

Comment:
BTW, as I'm trying to process in my mind. I am imagining that BytesWritableInputRowParser would have dependency on hadoop libs and that parser may not actually work (when hadoop libs are loaded via extension and not put explicitly on overlord classpath)
A part of my brain is now thinking, may be it is simplest to add special handling for popular stuff here for now.